src/vmspawn/vmspawn.c

```diff
@@ -30,6 +32,7 @@
 #include "sd-event.h"
 #include "signal-util.h"
 #include "socket-util.h"
+#include "string-util-fundamental.h"
```

Comment:
usually foo.h already imports foo-fundamental.h if such a pairing exists, and you sould only include foo.h hence.

or in other words, please change this to `#include "string-util.h"`, similar above

Reply:
Fixed in 37e75950ff.